lib/avalanche.ex

```diff
@@ -95,6 +117,16 @@ defmodule Avalanche do
                           required: false,
                           default: false,
                           doc: "Set to true to execute the statement asynchronously and return the statement handle."
+                        ],
+                        request_id: [
```

Comment:
Interesting. What's the use case exactly? Is the flow:
- the request cries
- has id in the response
- we retry because of it

Have you experienced this issue already in dev? Or, you are anticipating this (maybe it's common and you know it will happen)